clang/lib/CodeGen/CGCall.h

```diff
@@ -128,12 +136,12 @@ class CGCallee {
 
   static CGCallee forDirect(llvm::Constant *functionPtr,
                             const CGCalleeInfo &abstractInfo = CGCalleeInfo()) {
-    return CGCallee(abstractInfo, functionPtr);
+    return CGCallee(abstractInfo, functionPtr, CGPointerAuthInfo());
```

Comment:
Do we need to pass the default value manually? Can't we just leave this "as is"?

Reply:
Yes. I've fixed that.